extensions/federation/README.md

```diff
@@ -189,6 +189,18 @@ schema:
 
 **Note:** In Collections this should generally be provided on the top-level of the object.
 
+### Validation
+
+If this property is returned through the `POST /validation` endpoint, it has the meaning as listed below.
+This also covers the case where the federation supports splitting a process into pieces so that different parts can run on different back-ends.
+
+- Endpoint returns *without* errors:
+  - `federation:backends` is included in the response: The listed back-ends support the workflow (either partially if splitting is supported, or in full).
+  - `federation:backends` is *not* included in the response: At least one of the back-ends support the workflow.
+- Endpoint returns errors:
+  - `federation:backends` is included in the response: The listed back-ends were checked and none of the back-ends can run the workflow as is (neither splitted if supported, nor in full).
+  - `federation:backends` is *not* included in the response: None of the back-ends can run the workflow as is.
```

Comment:
I think the federation component itself (e.g. openeo-aggregator) could already raise validation issues, without any need to consult actual backends (e.g. if the whole post payload or process graph is completely nonsensical).

That would fall under "returns errors" + "federation:backends not included in response" I guess, but that might need a bit of clarification. e.g.

```suggestion
  - `federation:backends` is *not* included in the response: the workflow could not be validated successfully by any of the back-ends or the federation component itself.
```